docs/source/development/coding_conventions.rst

```diff
@@ -231,6 +231,56 @@ corresponds to complexity of the body).
 
 |x| **Do not** declare imports not used within the file.
 
+Element ID
+----------
```

Comment:
@jhopen Can we add a sentence or two to reflect why we are adding element ids? I feel that showing the reason and not just the rule will help guide devs... and of course we didn't write that yet

Here is the start of something, please let me know if you think it makes sense:

> To allow for easier integration testing of the MOTECH-UI, we are using a naming convention for the element IDs of navigational and action based items in the UI. The element IDs should follow the format below, unless there is a clear reason the format shouldn't be followed.